libcontainer/cgroups/fs/apply_raw.go

```diff
@@ -351,7 +351,11 @@ func writeFile(dir, file, data string) error {
 	if dir == "" {
 		return fmt.Errorf("no such directory for %s.", file)
 	}
-	return ioutil.WriteFile(filepath.Join(dir, file), []byte(data), 0700)
+	filePath := filepath.Join(dir, file)
+	if err := ioutil.WriteFile(filePath, []byte(data), 0700); err != nil {
+		return fmt.Errorf("failed to write %v to %v: %v", data, filePath, err)
```

Comment:
Since some of these filepaths can be long do you think we should just get the filename?

i.e.

/sys/fs/cgroup/memory/docker/ffbd278589d17535a00308ee65564b0947e389ce8fa4c26c4dfd3e52d834cb62/memory.limit_in_bytes

vs 

memory.limit_in_bytes